logging/Troubleshooting.md

```diff
@@ -100,13 +118,93 @@ INFO The Kibana REST endpoint does not appear to be quite ready [000]; sleeping
 The Kibana REST endpoint has NOT become accessible in the expected time; exiting.
 Review the Kibana pod's events and log to identify the issue and resolve it before trying again.
 ```
+
 This happens when Kibana cannot be accessed from the machine on which the deployment process is running via the same
-URL your end-users will be using.  This might the case if your environment has been configured to only allow access via
-a private network inaccessable to the deployment machine.  In these situations, setting the LOG_ALWAYS_PORT_FORWARD 
-property to "true" will force the deployment scripts to use Kubernetes port-forwarding when accessing the Kibana and/or 
+URL your end-users will be using.  This might the case if your environment has been configured to allow access only via
+a private network inaccessible to the deployment machine.  In these situations, setting the `LOG_ALWAYS_PORT_FORWARD` 
+property to `true` forces the deployment scripts to use Kubernetes port-forwarding when accessing the Kibana and/or 
 Elasticsearch APIs rather than the end-user URLs.
 
 ### Solution
-Set the LOG_ALWAYS_PORT_FORWARD environment variable to "true" before (re-)running the deployment script.  This should be done 
-by modifying (or adding) the appropriate line in the $USER_DIR/logging/user.env file.  See the [main README](../README.md#customization) 
+
+Set the `LOG_ALWAYS_PORT_FORWARD` environment variable to `true` before (re-)running the deployment script.  This should be done 
+by modifying (or adding) the appropriate line in the `$USER_DIR/logging/user.env` file.  See the [main README](../README.md#customization) 
 for information about the customization process and how to set up a USER_DIR.
+
+## Issue: Log Messages Require More Storage
+
+### Description
+
+You might need to increase storage in the following situations:
+
+* The persistent volume claims (PVCs) associated with the Elasticsearch nodes 
+(pods) that are used for storage (that is, the nodes with a role of "data") are 
+running low on space. 
+* You are considering an increase to how long log messages are retained. 
+Longer retention periods require more storage.
+ 
+**Tip:** In the default configuration, these pods have names that start with 
+"v4m-es-data-*". 
+
+### Solution
+
+**Note:** Some Kubernetes storageClass resources do not support expansion. To 
+determine whether the storageClass resources used by the Elasticsearch data node 
+PVCs support expansion, enter the following command: 
+
+`kubectl describe storageClass default` 
+
+* Where ***storageClass*** is the name of your specific storageClass.
+* Where ***default*** is the name of the storageClass that is used for the PVCs 
+that you are checking.  
+  
+If the `AllowVolumeExpansion` property is `True`, the expansion of existing PVCs 
```

Comment:
Well, that wasn't clear to *me*. Adding a sentence instead of a clause. I prefer to avoid run on sentences.

Review the command ouput. If the `AllowVolumeExpansion` property is `True`, ...